web/sites/default/config/core.entity_form_display.group.bureau_office.default.yml

```diff
@@ -33,12 +33,12 @@ dependencies:
     - field.field.group.bureau_office.field_group_menu_toggle
     - field.field.group.bureau_office.field_group_path
     - field.field.group.bureau_office.field_instagram
-    - field.field.group.bureau_office.field_linkedin
```

Comment:
@brittanysmart, please check the issue for some quick changes to make sure we don't lose content. After deploying those changes, this comment should show as outdated so you don't need to resolve the comment.